sources/packages/api/src/route-controllers/institution-locations/institution-locations.controller.ts

```diff
@@ -27,4 +32,26 @@ export class InstitutionLocationsController extends BaseController {
       data: location.data,
     };
   }
+
+  @Post()
+  async create(
+    @Body() payload: InstitutionLocationType,
```

Comment:
We are trying to keep all objects received/returned by the API as DTO. I would recommend moving this to the DTO level.